tracer/test/Datadog.Trace.Tests/DataStreamsMonitoring/DataStreamsMonitoringTransportTests.cs

```diff
@@ -85,7 +85,14 @@ public async Task TransportsWorkCorrectly(Enum transport)
              .OnlyContain(
                   headers => headers.AllKeys.Contains("Content-Encoding")
                           && headers["Content-Encoding"] == "gzip");
-        result.Should().OnlyContain(payload => payload.Stats.Count(s => s.Backlogs != null) == 1);
+
+        // should only have a single backlog across all payloads, but we don't know which payload it will be in
+        result
+           .SelectMany(x => x.Stats)
+           .Where(x => x.Backlogs is not null)
+           .Should()
+           .ContainSingle()
+           .And.ContainSingle();
```

Comment:
Oops

```suggestion
        // should only have a single backlog across all payloads, but we don't know which payload it will be in
        result
           .SelectMany(x => x.Stats)
           .Where(x => x.Backlogs is not null)
           .Should()
           .ContainSingle()
           .Which.Backlogs.Should()
           .ContainSingle();
```